parser/src/main/java/com/microsoft/gctoolkit/parser/jvm/UnifiedDiarizer.java

```diff
@@ -240,6 +236,11 @@ private void discoverJVMEvents(String line) {
 
     @Override
     public DateTimeStamp getTimeOfFirstEvent() {
-        return startTime;
+        return getDiary().getTimeOfFirstEvent();
+    }
+
+    @Override
+    public DateTimeStamp getTimeOfLastEvent() {
+        return null;
```

Comment:
Is it always null?

Reply:
not the best place to get this data and it's not really needed until we can get it from a better place so completely removed from the interface and implementing classes